designer/client/load-form-configurations.js

```diff
@@ -17,7 +17,7 @@ export function fetchConfigurations() {
 }
 
 export async function loadConfigurations() {
-  return fetchConfigurations()
+  return await fetchConfigurations()
```

Comment:
no need to action, but some info on returning an await - it's a tiny performance thing but I understand you've probably done this for debugging or handling the data where it's called. two promises end up having to be unwrapped here 
https://eslint.org/docs/rules/no-return-await